src/main/java/mate/academy/service/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.service.impl;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.Order;
+import mate.academy.model.ShoppingCart;
+import mate.academy.model.User;
+import mate.academy.service.OrderService;
+import mate.academy.service.ShoppingCartService;
+
+@Service
+public class OrderServiceImpl implements OrderService {
+    @Inject
+    private OrderDao orderDao;
+    @Inject
+    private ShoppingCartService shoppingCartService;
+
+    @Override
+    public Order completeOrder(ShoppingCart shoppingCart) {
+        User user = shoppingCart.getUser();
+        shoppingCartService.clearShoppingCart(shoppingCart);
+        LocalDateTime localDateTime = LocalDateTime.now();
+        return orderDao.add(new Order(shoppingCart.getTickets(),localDateTime,user));
+    }
```

Comment:
what will be in our order if we firstly clear shoppingCart and after that get tickets from shoppingCart to order?
also 
<img width="974" alt="image" src="https://user-images.githubusercontent.com/58032907/207279732-d0451210-42b2-4ebb-b216-8624a4d6ac45.png">